Library/Homebrew/tap.rb

```diff
@@ -112,6 +112,16 @@ def to_hash
     }
   end
 
+  def load_tap_formula_renames
+    begin
+      path.cd do
+        load "tap_formula_renames.rb"
```

Comment:
Let's use different name. It's easier, robust and more friendly to existing tools/scripts.